openstack/bms/v2/servers/get.go

```diff
@@ -0,0 +1,20 @@
+package servers
+
+import (
+	"github.com/opentelekomcloud/gophertelekomcloud"
+	"github.com/opentelekomcloud/gophertelekomcloud/internal/extract"
+)
+
+// Get requests details on a single server, by ID.
+func Get(client *golangsdk.ServiceClient, id string) (*Server, error) {
+	raw, err := client.Get(client.ServiceURL("servers", id), nil, &golangsdk.RequestOpts{
+		MoreHeaders: map[string]string{"X-OpenStack-Nova-API-Version": "2.26"},
```

Comment:
This is mandatory header?

Reply:
see https://docs.otc.t-systems.com/api/bms/en-us_topic_0171226371.html